### ISSUE TITLE
Question about fine tuning a AR model

### ISSUE THREAD

**Issue by GUUser91** [Plachtaa/seed-vc#161]
I tried a finetuning a AR model with the --train-ar flag. But the output file always starts speaking in gibberish. Is it because I need more data?

**Comment by Plachtaa** [comment]
Try not to fine tune AR model for now because I have the same finding as yours. It needs more research

**Comment by ABC0408** [comment]
If I want to fine-tune on singing voice data, should both train_cfm and train_ar be enabled, or is it recommended to train from scratch?
